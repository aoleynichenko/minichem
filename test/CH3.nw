start CH3
echo

geometry
C     0.08745162    -0.08744725    -0.08742186
H     0.52503428     0.78909888    -0.52508604
H    -0.78884450    -0.52530342    -0.52536318
H     0.52530257    -0.52529218     0.78892712
end

# STO-3G
basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
     0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  uhf
#  nodiis
#  direct
  doublet
end

task scf


