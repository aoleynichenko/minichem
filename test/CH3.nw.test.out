Minichem 1.0
By Alexander Oleynichenko, 2018
Build date:    Jan  2 2019 22:39:09
Compiler:      gcc 4.9.2
Date:          Fri Jan  4 14:00:30 2019
# MPI threads: 1
Arguments:  minichem.x CH3.nw 
------------------------------------------------------------
>>> CH3.nw
start CH3
echo

geometry
C     0.08745162    -0.08744725    -0.08742186
H     0.52503428     0.78909888    -0.52508604
H    -0.78884450    -0.52530342    -0.52536318
H     0.52530257    -0.52529218     0.78892712
end

# STO-3G
basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
     0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  uhf
  doublet
  nodiis
end

task scf


------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | ao basis        |   1    |     3      |
|   C     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------

		AO integrals evaluation module
		------------------------------

Output files:
  AOINTS1   one-electron integrals
  AOINTS2   two-electron integrals
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika

	Atomic coordinates (angstroms):
    [ 1]  C      0.08745161    -0.08744724    -0.08742185
    [ 2]  H      0.52503424     0.78909882    -0.52508600
    [ 3]  H     -0.78884444    -0.52530338    -0.52536314
    [ 4]  H      0.52530253    -0.52529214     0.78892706


	Distance matrix (angstroms):
    [ 1] C     [ 2] H      1.07301513
    [ 1] C     [ 3] H      1.07303558
    [ 1] C     [ 4] H      1.07303728
    [ 2] H     [ 3] H      1.85847530
    [ 2] H     [ 4] H      1.85856235
    [ 3] H     [ 4] H      1.85858576

generating atom-centered basis set...
  # bfns   = 8
  # shells = 6

	Basis functions (atomic units):
    C    0.165  -0.165  -0.165  [1]s  [2]s  [3]pz  [4]py  [5]px
    H    0.992   1.491  -0.992  [6]s
    H   -1.491  -0.993  -0.993  [7]s
    H    0.993  -0.993   1.491  [8]s

begin 1e integrals...
  overlap
  kinetic energy
  nuclear attraction
  X Y Z integrals
  XX XY XZ YY YZ ZZ integrals
    center: origin = 0.0 0.0 0.0
done
begin 2e integrals...
  sizeof buf (bytes) = 98304
  sizeof integral (bytes) = 24
  # unique ERIs   = 666
  # non-zero ERIs = 579
done

------------------------------------------------------------

          ********************************
          *      Parallel SCF Module     *
          ********************************

                    SCF Parameters
                    --------------
               print level : medium
              wavefunction : UHF
            max iterations : 50
              density conv : 1e-06
               energy conv : 1e-07
           integral direct : off
                      diis : off


Atoms:     4
Electrons: 0
Mol mass:  0.000
Charge:    0
Spin mult: 2
Units:     angstroms
One-electron integrals read in 0.000000 sec
Basis orthogonalization algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.25052657
AO basis orthogonalization done in 0.000171 sec

Initial guess: extended Huckel theory
Initial energy = -48.56977360

Nalpha = 5
Nbeta = 4

 iter.       Energy         Delta E       RMS-Dens      <S^2>       time
--------------------------------------------------------------------------
   1     -39.43862546    -0.59994813     0.22085043    0.7529       0.00
                                         0.16942407
   2     -39.04517277     0.39345270     0.04503756    0.7564       0.00
                                         0.04097303
   3     -39.07578728    -0.03061452     0.01544750    0.7592       0.00
                                         0.01261844
   4     -39.07565891     0.00012838     0.00639313    0.7611       0.00
                                         0.00604319
   5     -39.07620997    -0.00055106     0.00356202    0.7624       0.00
                                         0.00348265
   6     -39.07640437    -0.00019441     0.00218267    0.7633       0.00
                                         0.00215361
   7     -39.07650334    -0.00009897     0.00136883    0.7638       0.00
                                         0.00135187
   8     -39.07655802    -0.00005468     0.00086255    0.7642       0.00
                                         0.00085183
   9     -39.07659028    -0.00003227     0.00054413    0.7644       0.00
                                         0.00053731
  10     -39.07660995    -0.00001967     0.00034336    0.7646       0.00
                                         0.00033904
  11     -39.07662214    -0.00001219     0.00021669    0.7647       0.00
                                         0.00021395
  12     -39.07662976    -0.00000762     0.00013675    0.7647       0.00
                                         0.00013502
  13     -39.07663454    -0.00000478     0.00008630    0.7648       0.00
                                         0.00008521
  14     -39.07663755    -0.00000301     0.00005447    0.7648       0.00
                                         0.00005378
  15     -39.07663945    -0.00000189     0.00003437    0.7648       0.00
                                         0.00003394
  16     -39.07664064    -0.00000119     0.00002169    0.7648       0.00
                                         0.00002142
  17     -39.07664140    -0.00000075     0.00001369    0.7648       0.00
                                         0.00001352
  18     -39.07664187    -0.00000048     0.00000864    0.7648       0.00
                                         0.00000853
  19     -39.07664217    -0.00000030     0.00000545    0.7648       0.00
                                         0.00000538
  20     -39.07664236    -0.00000019     0.00000344    0.7648       0.00
                                         0.00000340
  21     -39.07664248    -0.00000012     0.00000217    0.7648       0.00
                                         0.00000214
  22     -39.07664255    -0.00000008     0.00000137    0.7648       0.00
                                         0.00000135
  23     -39.07664260    -0.00000005     0.00000086    0.7648       0.00
                                         0.00000085
--------------------------------------------------------------------------
          Total SCF energy =   -39.07664260
  Nuclear repulsion energy =     9.73109626
                 UHF <S^2> =     0.7648
               Exact <S^2> =     0.7500

      Alpha Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  1  |   -11.00548391 |     ?    |
  | 2   |  1  |    -0.90863571 |     ?    |
  | 3   |  1  |    -0.54118757 |     ?    |
  | 4   |  1  |    -0.54117012 |     ?    |
  | 5   |  1  |    -0.35742212 |     ?    |
  | 6   |  0  |     0.63812344 |     ?    |
  | 7   |  0  |     0.77493254 |     ?    |
  | 8   |  0  |     0.77497627 |     ?    |
  +-----+-----+----------------+----------+

       Beta Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  1  |   -10.97427236 |     ?    |
  | 2   |  1  |    -0.79922633 |     ?    |
  | 3   |  1  |    -0.52410261 |     ?    |
  | 4   |  1  |    -0.52408317 |     ?    |
  | 5   |  0  |     0.33346962 |     ?    |
  | 6   |  0  |     0.70177343 |     ?    |
  | 7   |  0  |     0.80148252 |     ?    |
  | 8   |  0  |     0.80152607 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  C     6.17943605  -0.17943605
      2  H     0.94018921   0.05981079
      3  H     0.94019103   0.05980897
      4  H     0.94018371   0.05981629


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  C     6.09533547  -0.09533547
      2  H     0.96822355   0.03177645
      3  H     0.96822388   0.03177612
      4  H     0.96821709   0.03178291


		SCF properties
		--------------

  moments are calculated at point (0,0,0)

  dipole moment:
  D       x            y            z
    -0.00013130   0.00013049   0.00013535
  |D| = 0.00022932 a.u. = 0.00058288 Debye

  second moments of charge (atomic units):
           xx           yy           zz
      -5.236015    -5.235812    -5.235882
           xy           xz           yz
       0.181349     0.181549    -0.181421
  quadrupole moments (atomic units):
           xx           yy           zz
      -0.000337     0.000274     0.000063
           xy           xz           yz
       0.544048     0.544647    -0.544263
  Q_ij = 3*r_i*r_j - \delta_ij*|r^2|
  1 Buckingham = 1.345044 a.u.


 time for (sec):
 -------------------------------------------------------
  One-electron integrals                          0.005
  Two-electron integrals                          0.049
  Basis set orthogonalization                     0.000
  Initial guess                                   0.000
  Fock matrix diagonalization                     0.001
  Density matrix construction                     0.000
  Fock matrix construction                        0.002
 -------------------------------------------------------
                                                  0.058


Allocated: 228224 bytes
Freed:     204672 bytes
Max usage: 34944 bytes
Total time: 0.078 sec
